ArduCopter/Copter.cpp

```diff
@@ -481,6 +481,9 @@ void Copter::fourhundred_hz_logging()
 {
     if (should_log(MASK_LOG_ATTITUDE_FAST) && !copter.flightmode->logs_attitude()) {
         Log_Write_Attitude();
+#if CUSTOMCONTROL_ENABLED == ENABLED && FRAME_CONFIG != HELI_FRAME
+        attitude_control->log_custom_controller();
```

Comment:
I would be tempted to not bother with the defines here. You have the function in the base class. You could probably just call it `log` or `log_write`

Reply:
Do you mean "#if CUSTOMCONTROL_ENABLED == ENABLED && FRAME_CONFIG != HELI_FRAME
" can be removed with no effect?

Reply:
Yeah, the function exists in both controllers. So you can just call it all the time. It will be a minimal flash cost. Of course in this case calling it `log_custom_controller` is not correct, it becomes a generic attitude control log function.